### PR TITLE
Vd photogram pundit

### PR DESCRIPTION
**Pull request by viviancreates**

### CODE REVIEW

File: app/controllers/photos_controller.rb
Comment (dsimp) [inline app/controllers/photos_controller.rb:1]: Nice way of implementing the authorize method

**Review by dsimp — APPROVED** [review]
Nice conditional statements and implementing the authorizations

**Review by tluu5 — COMMENTED** [review]
great job adding the except and only on follow_requests / likes

**Review by tluu5 — COMMENTED** [review]
like that you added auth for users controller

**Review by tluu5 — APPROVED** [review]
You did a great job using policy scope.

**Pull request closed without merging**